README.org

```diff
@@ -109,41 +109,42 @@
+   | Ruby                  | [[https://github.com/castwide/solargraph][solargraph]]                                | Yes            | gem install solargraph                                                                                                                                   | Yes                          |
+   | Rust                  | [[https://github.com/rust-lang-nursery/rls][rls]]                                       | Yes            | [[https://github.com/rust-lang-nursery/rls][rls]]                                                                                                                                                      | Yes                          |
+   | Scala                 | [[https://github.com/rossabaker/lsp-scala][lsp-scala]]                                 | [[https://github.com/rossabaker/lsp-scala][lsp-scala]]      | [[https://github.com/rossabaker/lsp-scala][lsp-scala]]                                                                                                                                                |                              |
+   | Swift                 | [[https://github.com/apple/sourcekit-lsp][sourcekit-LSP]]                             | [[https://github.com/emacs-lsp/lsp-sourcekit][lsp-sourcekit]]  | [[https://github.com/apple/sourcekit-lsp][sourcekit-LSP]]                                                                                                                                            | Yes (via llvm debug adapter) |
+   | Vue                   | [[https://github.com/vuejs/vetur/tree/master/server][vue-language-server]]                       | Yes            | npm install -g vue-language-server                                                                                                                       | Yes (Firefox/Chrome)         |
+   | XML                   | [[https://github.com/angelozerr/lsp4xml][lsp4xml]]                                   | Yes            | Download from [[https://github.com/angelozerr/lsp4xml/releases][lsp4xml releases]]                                                                                                                           |                              |
+   | Elm                   | [[https://github.com/elm-tooling/elm-language-server][elmLS]]                                     | Yes            | Clone the [[https://github.com/elm-tooling/elm-language-server][repository]], run npm install in the root, then run [[https://www.typescriptlang.org][tsc]] in the server directory. Set lsp-elm-server-install-dir to wherever you cloned the repo. | No                           |
```

Comment:
Can you move it on the proper place (the languages are sorted alphabetically)?

Reply:
Think this is in the right place now. Sorry I didn't even think about sorting lol